Sources/tart/Platform/Darwin.swift

```diff
@@ -58,7 +64,11 @@ struct UnsupportedHostOSError: Error, CustomStringConvertible {
       VZMacOSBootLoader()
     }
 
-    func platform(nvramURL: URL) throws -> VZPlatformConfiguration {
+    func platform(nvramURL: URL, needsNestedVirtualization: Bool) throws -> VZPlatformConfiguration {
+      if needsNestedVirtualization {
+        throw UnsupportedNestedVirtualization()
```

Comment:
What do you think about throwing `RuntimeError.VMConfigurationError` here with an appropriate message, instead of introducing a new `Error` struct?